src/test/java/org/cactoos/proc/BiProcOfTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2020 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.proc;
+
+import java.util.concurrent.atomic.AtomicReference;
+import org.cactoos.func.BiFuncOf;
+import org.cactoos.func.FuncOf;
+import org.junit.jupiter.api.Test;
+import org.llorllale.cactoos.matchers.Assertion;
+import org.llorllale.cactoos.matchers.MatcherOf;
+
+/**
+ * Test case for {@link BiProcOf}.
+ *
+ * @since 0.49
+ */
+public class BiProcOfTest {
+    @Test
+    void worksWithFunc() throws Exception {
+        final AtomicReference<Object> done = new AtomicReference<>();
+        new Assertion<>(
+            "Must execute BiProc with Func",
+            new BiProcOf<>(
+                new FuncOf<>(
+                    input -> {
+                        done.set(input);
+                        return new Object();
```

Comment:
@baudoliver7 since the output is going to be ignored in practice, let's make this obvious by using the string "discarded" here, as in the `RunnableOfTest` class for example. Same everywhere in those tests wherever it makes sense

Reply:
@victornoel Ok